src/controllers/chapters_controller.ts

```diff
@@ -2,42 +2,62 @@ import { Request, Response } from "express";
 import { ChapterService } from "../services/chapters_service";
 import { PostgresChapterDAO } from "../dao/postgres_impl/postgres_chapters_dao";
 import { IChapter } from "../models/chapter";
+import { areKeysNotValid, errorMessage } from "../utils/helpers";
 
 export class ChaptersController {
      service: ChapterService;
-
+     validKeys: string[] = ["topic_id", "title", "description"];
+     erMsg: string[] = [
+          "The request body is empty. Please put input values.",
+          "Error while updating the chapter. Please check your input values or try again later.",
+          "One of the field is not appropriate. The valid keys are : [topic_id, title, description].",
+          "Error while creating the chapter. Please check your input values or try agin later",
+          "There is no chapter with the given id. Please check your input values or try again later.",
+          "There is no chapter yet !",
+          "Error while retrieving chapters. Please try again",
+     ];
```

Comment:
Isn't there a way to do it by using enum ?
I don't know if this exists in javascript, but in rust we have something like:
```rust
enum ErrorMessage {
     EmptyRequestBody(String),
     ErrorWhileUpdatingChapter(String),
    ....
}
```

Reply:
If there is no way to do it via enum, then that's fine

Reply:
You're right
There is a more efficient way by using enum .

Reply:
Nice. I think it will be easier to maintain in the long term